### PR TITLE
Find add playground card

### PR DESCRIPTION
**Pull request by KarafiziArtur**
https://github.com/local-motion/product/issues/331

### CODE REVIEW

**Comment by maxvisser** [comment]
Hi Arthur, great start! Is it possible to only show dutch placenames in search results?

File: src/views/Workspace/Cards/AddFindPlayground.jsx [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+        const { classes, googleMapsKey } = this.props;
+        const { isAddPlaygroundOpen, userAddress } = this.state;
+
+        return (
+          <div>
+              <WorkspaceCard
+                title={"Speeltuin toevoegen"}
+                done={null}
+                image={require("../../../assets/img/backgrounds/workspace-welcome.jpg")}
+                content={"De eerste stap is om je aan te sluiten bij de actiepagina van jouw speeltuin, of om deze aan te maken. Vul je gegevens in om te beginnen."}
```
Comment (maxvisser) [inline src/views/Workspace/Cards/AddFindPlayground.jsx]: De eerste stap is om te kijken of je je kunt aansluiten bij een bestaande actiepagina van jouw speeltuin. Misschien is een team bezig om jou speeltuin rookvrij te maken. Vul je plaatsnaam, adres of postcode in om te beginnen.

File: src/views/Workspace/Cards/AddFindPlayground.jsx [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+                              getResults={this.getResults}
+                              classes={classes}
+                              googleMapsKey={googleMapsKey}
+                            />
+
+                            <Button
+                              className={classes.gotoMapButton}
+                              onClick={this.toggleAddPlayground}
+                            >
+                                Of zoek op de kaart
```
Comment (maxvisser) [inline src/views/Workspace/Cards/AddFindPlayground.jsx]: this was not meant to add a playground; only for searching on map. A button can be below the map to add a playground,

File: src/views/Workspace/Cards/AddFindPlayground.jsx [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+    }
+
+    render() {
+        const { classes, googleMapsKey } = this.props;
+        const { isAddPlaygroundOpen, userAddress } = this.state;
+
+        return (
+          <div>
+              <WorkspaceCard
+                title={"Speeltuin toevoegen"}
```
[the remark this answers is not in the document]
Comment (KarafiziArtur) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Done!

[the remark this answers is not in the document]
Comment (maxvisser) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: hmm. Lets do 'Actie starten' instead. Makes it more aligned with CTA on frontpage

[the remark this answers is not in the document]
Comment (maxvisser) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Lets also change menu item to 'Actie starten' (currently also 'speeltuin toevoegen'

File: src/views/Workspace/Cards/AddFindPlayground.jsx (hunk 1 above)
Comment (KarafiziArtur) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Done

File: src/views/Workspace/Cards/AddFindPlayground.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (KarafiziArtur) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Ok, I got. Done

File: src/views/Workspace/Cards/AddFindPlayground.jsx (hunk 2 above)
Comment (maxvisser) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: made issue for this; this maybe needs beta testing, on how people use this.

**Comment by maxvisser** [comment]
Lets move all changes to google map functionality to a seperate branch / pr; this is ok for now

File: src/views/Workspace/Cards/AddFindPlayground.jsx [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+                    focused: props.classes.cssFocused,
+                }
+            }}
+            InputProps={{
+                endAdornment:
+                  <InputAdornment disablePointerEvents position="end">
+                      <Search color="disabled" />
+                  </InputAdornment>,
+            }}
+            inputProps={{
```
Comment (dimitriirtimid) [inline src/views/Workspace/Cards/AddFindPlayground.jsx]: this duplicate the inputProps properties which generates a warning (and thus blocks the build).
Please fix.

Comment (KarafiziArtur) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Oh, actually both of them (with a different case of the first letter) is important for the correct displaying. But I'll try to find a solution.

File: src/views/Onboarding/components/GooglePlacesAutocomplete.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    }}
+  >
+      {({ getInputProps, suggestions, getSuggestionItemProps }) => (
+        <div>
+            <TextField
+              type="text"
+              variant="outlined"
+              name="search"
+              className={props.classes.textInput}
+              label="Zoek in je plaats"
```
Comment (maxvisser) [inline src/views/Onboarding/components/GooglePlacesAutocomplete.jsx]: label should be "Zoek naar speeltuinen bij je in de buurt"

**Comment by maxvisser** [comment]
Looking at UI these elements can be combined; I think 'Of zoek op kaart' can just be replaced by the map toggle.

image

File: src/views/Workspace/Cards/AddFindPlayground.jsx [hunk 6]
```diff
@@ -0,0 +1,504 @@
...
+                                <div className={classes.resultSubtitle}>{name}</div>
+                            </div>
+
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Afstand</div>
+                                <div className={classes.resultSubtitle}>{convertedDistance}</div>
+                            </div>
+
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Deelneemers</div>
```
Comment (maxvisser) [inline src/views/Workspace/Cards/AddFindPlayground.jsx]: Deelnemers

Comment (KarafiziArtur) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Done

File: src/views/Onboarding/components/GooglePlacesAutocomplete.jsx (hunk 5 above)
Comment (KarafiziArtur) [reply src/views/Onboarding/components/GooglePlacesAutocomplete.jsx]: Done

File: src/views/Workspace/Cards/AddFindPlayground.jsx (hunk 4 above)
Comment (KarafiziArtur) [reply src/views/Workspace/Cards/AddFindPlayground.jsx]: Done

**Comment by maxvisser** [comment]
@dimitriirtimid good catch!

**Comment by dimitriirtimid** [comment]
merged into master. 
Can we close this PR?

**Comment by maxvisser** [comment]
yes

**Pull request closed without merging**